tests/happy.rs

```diff
@@ -23,8 +23,8 @@ enum Happy {
      * Lets see how this works out for it
      */
     Variant4,
-    /// Variant5 just has {0} many problems
-    /// but multi line comments aren't one of them
+    /// Variant5 has a parameter {0} and some regular comments
+    // A regular comment that won't get picked
```

Comment:
I'm sorry for not mentioning this first, but instead of making the second comment here into a regular comment, can we just remove this variant from this test file and instead add an equivalent compile test that asserts that we panic with the expected error when there are multiple doc attributes?

Reply:
Sure, I'll give it a shot. I'm running into issues running the tests even on `master`.